<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Android File Transfer for Linux</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="listView">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionRename"/>
    <addaction name="actionDownload"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionCreateDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuNavigate">
    <property name="title">
     <string>&amp;Navigate</string>
    </property>
    <addaction name="actionBack"/>
    <addaction name="actionGo_Down"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNavigate"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionGo_Down"/>
   <addaction name="actionCreateDirectory"/>
   <addaction name="actionUpload_Album"/>
   <addaction name="actionUploadDirectory"/>
   <addaction name="actionUpload"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUpload">
   <property name="icon">
    <iconset>
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </action>
  <action name="actionCreateDirectory">
   <property name="icon">
    <iconset theme="folder-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>CreateDirectory</string>
   </property>
  </action>
  <action name="actionUploadDirectory">
   <property name="text">
    <string>UploadDirectory</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset theme="go-previous">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Back</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionGo_Down">
   <property name="icon">
    <iconset theme="go-next">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Go Down</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionUpload_Album">
   <property name="text">
    <string>Upload Album</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset theme="edit-rename">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Rename</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>D&amp;elete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset theme="download">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Download</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
